### PR TITLE
GAD-1: fixed missing template issue and added data of alumni and achievement

### PR DESCRIPTION
**Pull request by CHOUHAN-MOHIT**
changed the template name from DESIGN_dep.html to design_dep.html.

### CODE REVIEW

**Review by Sha2nkGitHub — COMMENTED** [review]
ping @manavdesai27 @akshatnema

**Comment by Sha2nkGitHub** [comment]
ping @akshatnema @manavdesai27

File: FusionIIIT/templates/department/cse_dep.html [hunk 1]
```diff
@@ -414,6 +414,19 @@ <h1><center> Welcome to CSE Department </center></h1>
         <div class="ui vertical segment">
             <div class="content">
                 <div class="description">
+                    <a class="active item" href="./All_Students/{{1111}}">
+                        <!--   <button style="border: none; float: left; display: block;" class="ui inverted right floated small red button" onclick="requestModal()"> -->
```
Comment (akshatnema) [inline FusionIIIT/templates/department/cse_dep.html]: Remove these type of comments from your PR File changes.

Comment (CHOUHAN-MOHIT) [reply FusionIIIT/templates/department/cse_dep.html]: @akshatnema Necessary changes have been made. Please check it.

File: FusionIIIT/templates/department/cse_dep.html [hunk 2]
```diff
@@ -260,13 +260,13 @@ <h1><center> Welcome to CSE Department </center></h1>
                             </div>
                             <div class="content">
                                 <a class="item" href="./All_Students/{{11111}}">
-                                    <!--   <button style="border: none; float: left; display: block;" class="ui inverted right floated small red button" onclick="requestModal()"> -->
+                                    {% comment %}    <button style="border: none; float: left; display: block;" class="ui inverted right floated small red button" onclick="requestModal()">  {% endcomment %}
```
Comment (akshatnema) [inline FusionIIIT/templates/department/cse_dep.html]: Kindly remove these comments from the PR, since they are no longer needed.

Comment (akshatnema) [inline FusionIIIT/templates/department/cse_dep.html]: Kindly remove these comments from the PR, since they are no longer needed.

Comment (CHOUHAN-MOHIT) [reply FusionIIIT/templates/department/cse_dep.html]: remove all unnecessary comments.

**Comment by CHOUHAN-MOHIT** [comment]
@akshatnema

File: FusionIIIT/templates/department/design_dep.html [hunk 3]
```diff
@@ -231,7 +257,7 @@ <h3>
                             </div>
                             <div class="content">
                                 <a class="item" href="./All_Students/{{1111111}}">
-                                    <!--   <button style="border: none; float: left; display: block;" class="ui inverted right floated small red button" onclick="requestModal()"> -->
+                                    {% comment %}    <button style="border: none; float: left; display: block;" class="ui inverted right floated small red button" onclick="requestModal()">  {% endcomment %}
```
Comment (akshatnema) [inline FusionIIIT/templates/department/design_dep.html]:
```suggestion
```
Remove these type of comments from this file.

**Review by akshatnema — CHANGES_REQUESTED** [review]

**Review by akshatnema — APPROVED** [review]

**Pull request merged**